plugins/env/README.md

```diff
@@ -1,4 +1,39 @@
+## Env Plugin
 Plugin to manage ENV settings for projects and write .env files during deploy.
 
 Includes `/projects/:permalink/environment?deploy_group=permalink` endpoint that returns the .env content
 for a project and deploy_group.
+
+## GitHub to manage environment variables
+This plugin has an option to use GitHub repository for .env files.  Using GitHub provides better control
+of the environment variables applied to each deploy group and provides the ability to have peer reveiw of 
```

Comment:
`provides better control` is kinda vague ... what does `better` mean ? it's slower but more flexible ... maybe just leave that out, does not provide any value

Reply:
Good comment ... I'll be more specific it supports the Pull Request process.

Reply:
I'll just leave it out ... if we need to add motivation for the feature we can add it to the document.